ui/src/app/components/buttons.tsx

```diff
@@ -427,6 +441,7 @@ export const IconButton = ({
   hover = {},
   tooltip = '',
   disabled = false,
+  title = '',
```

Comment:
This allows us to pass titles to icons in icon buttons.This helps accessibility and testing.